api/graphql/schema.graphql

```diff
@@ -2032,7 +2035,7 @@ type Mutation {
   # Remove a Post from a Collection, if you have permission to
   removePostFromCollection(collectionId: ID, postId: ID): GenericResult
   # Remove a badge/role from a member of a group, inititated by that member or by a group moderator
```

Comment:
oh interesting, you are saying someone can remove themself from a role? i'm actually not sure about that, if roles and badges are assigned by moderators they probably shouldnt be able to be removed by the person, only by moderators. a product question for the group

Reply:
Yeah. Currently its only implemented in the API, not the front-end